spec/validation/checksums.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+require "spec_helper"
+require_relative "../../lib/ht_sip_validator/sip/sip"
+require_relative "../../lib/ht_sip_validator/validation/checksums"
```

Comment:
Pushed a new commit to clean this up and ensure that the specs actually get run by default w/ `rspec`